compiler/rustc_middle/src/query/mod.rs

```diff
@@ -1594,6 +1594,11 @@ rustc_queries! {
     query object_lifetime_defaults(_: LocalDefId) -> Option<&'tcx [ObjectLifetimeDefault]> {
         desc { "looking up lifetime defaults for a region on an item" }
     }
+    /// Fetch the lifetimes for all the trait objects in an item-like.
+    query object_lifetime_map(_: LocalDefId) -> FxHashMap<ItemLocalId, Region> {
+        storage(ArenaCacheSelector<'tcx>)
+        desc { "looking up lifetime defaults for a region on an item" }
```

Comment:
Is this description correct?

It's probably worth elaborating a bit in a comment what the different between `object_lifetime_map` and `object_lifetime_defaults` are.